CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,64 @@
+# Code of Conduct
+
+Like the technical community as a whole, the various ESLint teams and communities are made up of a mixture of professionals and volunteers from all over the world, working on every aspect of the mission - including mentorship, teaching, and connecting people.
```

Comment:
I think this doesn't apply to us. We are all volunteers. "Professionals" in this context makes me think that there are people in the community that are getting paid to mentor, teach and connect people in ESLint community. This might be true in jQuery foundation, but we don't have a foundation.